ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.cpp

```diff
@@ -98,6 +98,11 @@ void BrowseOGCAPIFeatureService::handleError(const Esri::ArcGISRuntime::Error& e
 
 void BrowseOGCAPIFeatureService::loadFeatureService(const QUrl& url)
 {
+  if (m_featureService && m_featureService->loadStatus() == LoadStatus::Loading)
+  {
+    return;
+  }
```

Comment:
our coding standard doesn't require brackets for one line bodys

```suggestion
  if (m_featureService && m_featureService->loadStatus() == LoadStatus::Loading)
    return;
```